script/foreman-debug

```diff
@@ -246,9 +246,18 @@ else
 fi
 printv "Determined $OS distribution"
 
+clean_temp() {
+  [ "$NOTAR" -eq 1 ] && exit 0
+  # prevent from removing anything user-defined
```

Comment:
I wonder if we should delete the whole `/var/tmp` vs. `/var/tmp/foreman-debug-*`

Reply:
Sorry what?

Reply:
My goal here is to doublecheck user did not input "/" that could lead to disaster.

Reply:
Sry, read that wrong. Scratch that

Reply:
Thanks can you unblock the review then?

Reply:
Reading this again, it would actually delete user-defined thing, if that was in `/var/tmp`

Reply:
Scratch my previous comments. Anyway, one thing: you're calling `exit` that affects the exit code of the whole script: return should be used instead I guess.